docs/concepts/hyperparameters.md

```diff
@@ -1,5 +1,32 @@
 (hyperparameters)=
 # {octicon}`tools` Hyperparameters
 
-Explain the influence of number of epochs, batch size, for learning rate - reference to optimizer documentation
+Several of the `finetuner.fit` function's parameters are _hyper-parameters_: values that do not change the methods used during training but control the overall behavior and settings of the training process.
+These fall into four areas:
+- [Setting the learning rate and using an optimizer.](./optimizers.md)
+- [Setting a loss function and using a miner.](./loss-functions.md)
+- Setting the batch size.
+- Choosing how many epochs to train.
+
+For the first two, see the pages linked above. For epochs and batch sizes, read the discussion below.
+
+## Epochs
+An epoch is a single round of training in which the model is presented with each item of training data,
+and after each item, weights are updated.
```

Comment:
```suggestion
and during which the model's weights are updated multiple times.
```
the weights are not updated after processing each item, but only after processing a whole batch. If you use gradient accumulation (might be supported in the future) even more rarely.